src/upload.ts

```diff
@@ -34,15 +35,19 @@ const PORTAL_DIRECTORY_FILE_FIELD_NAME = "files[]";
  * @property [endpointLargeUpload] - The relative URL path of the portal endpoint to contact for large uploads.
  * @property [customFilename] - The custom filename to use when uploading files.
  * @property [largeFileSize=41943040] - The size at which files are considered "large" and will be uploaded using the tus resumable upload protocol. This is the size of one chunk by default (40 mib).
+ * @property [errorPages] - Defines a mapping of error codes and subfiles which are to be served in case we are serving the respective error code. All subfiles referred like this must be defined with absolute paths and must exist.
  * @property [retryDelays=[0, 5_000, 15_000, 60_000, 300_000, 600_000]] - An array or undefined, indicating how many milliseconds should pass before the next attempt to uploading will be started after the transfer has been interrupted. The array's length indicates the maximum number of attempts.
+ * @property [tryFiles] - Allows us to set a list of potential subfiles to return in case the requested one does not exist or is a directory. Those subfiles might be listed with relative or absolute paths. If the path is absolute the files must exist.
```

Comment:
```suggestion
 * @property [tryFiles] - Allows us to set a list of potential subfiles to return in case the requested one does not exist or is a directory. Those subfiles might be listed with relative or absolute paths. If the path is absolute the file must exist.
```

Reply:
Will pull into a follow-up since you approved the PR - note, however, that this was a copy/paste from the `skyd` repo on GitLabs.

Reply:
I don't think it's worth it to create a new PR just for this. Make the change - I'll re-approve.
I'll fix the typo in skyd as well.